Not a JSON Array: null

<!-- Please fill out the following information to help us in analyzing the issue, but feel free to skip it if you don't have the information at hand or if it does not apply. Please remove everything which does not apply to the issue. -->

### Description

### Info

* docker-maven-plugin version : 0.43.0
* Maven version (`mvn -v`) : 3.9.3
* Docker version : Docker version 24.0.0, build 98fdcd7

debug info
```
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.fabric8:docker-maven-plugin:0.43.0:build (docker-build) on project start-web-public: Execution docker-build of goal io.fabric8:docker-maven-plugin:0.43.0:build failed: Not a JSON Array: null
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:375)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:53)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:152)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:76)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution docker-build of goal io.fabric8:docker-maven-plugin:0.43.0:build failed: Not a JSON Array: null
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:148)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:53)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:152)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:76)
Caused by: java.lang.IllegalStateException: Not a JSON Array: null
    at com.google.gson.JsonElement.getAsJsonArray (JsonElement.java:107)
    at io.fabric8.maven.docker.access.hc.DockerAccessWithHcClient.getImageTags (DockerAccessWithHcClient.java:484)
    at io.fabric8.maven.docker.service.BuildService.removeDanglingImage (BuildService.java:455)
    at io.fabric8.maven.docker.service.BuildService.tagImage (BuildService.java:203)
    at io.fabric8.maven.docker.service.BuildService.tagImage (BuildService.java:125)
    at io.fabric8.maven.docker.BuildMojo.proceedWithDockerBuild (BuildMojo.java:109)
    at io.fabric8.maven.docker.BuildMojo.proceedWithBuildProcess (BuildMojo.java:90)
    at io.fabric8.maven.docker.BuildMojo.buildAndTag (BuildMojo.java:83)
    at io.fabric8.maven.docker.BuildMojo.processImageConfig (BuildMojo.java:175)
    at io.fabric8.maven.docker.BuildMojo.executeInternal (BuildMojo.java:72)
    at io.fabric8.maven.docker.AbstractDockerMojo.execute (AbstractDockerMojo.java:277)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:53)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:152)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:76)
```

my 'docker inspect' info below
```
    {
        "Id": "sha256:xxxxxx",
        "RepoTags": null,
        "RepoDigests": [
            "xxxxxx"
        ],
        "Parent": "sha256:xxxx",
        "Comment": "",
.....

```

this 'RepoTags' is null,
but there is not check null in `DockerAccessWithHcClient.java`,
```
 ... ...
    @Override
    public List<String> getImageTags(String name) throws DockerAccessException {
        HttpBodyAndStatus response = inspectImage(name);
        if (response.getStatusCode() == HTTP_NOT_FOUND) {
            return null;
        }
        JsonObject imageDetails = JsonFactory.newJsonObject(response.getBody());
        JsonArray tagsArr = imageDetails.get("RepoTags").getAsJsonArray();
        if (tagsArr.size() == 0) {
            return Collections.emptyList();
        }
        List<String> tags = new ArrayList<>();
        tagsArr.forEach(tag-> tags.add(tag.getAsString()));
        return tags;
    }
... ...
```

so, optimaize like this
```
    @Override
    public List<String> getImageTags(String name) throws DockerAccessException {
        HttpBodyAndStatus response = inspectImage(name);
        if (response.getStatusCode() == HTTP_NOT_FOUND) {
            return null;
        }
        JsonObject imageDetails = JsonFactory.newJsonObject(response.getBody());

        JsonElement repoTags = imageDetails.get("RepoTags"); // check json null
        if (repoTags.isJsonNull()) {
            return Collections.emptyList();
        }
        JsonArray tagsArr = repoTags.getAsJsonArray();
        if (tagsArr.size() == 0) {
            return Collections.emptyList();
        }
        List<String> tags = new ArrayList<>();
        tagsArr.forEach(tag-> tags.add(tag.getAsString()));
        return tags;
    }

```


* If it's a feature request, what is your use case :
* Sample project : *[GitHub Clone URL]*

--- a/src/main/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClient.java
+++ b/src/main/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClient.java
@@ -1,7 +1,5 @@
 package io.fabric8.maven.docker.access.hc;
 
-import static java.net.HttpURLConnection.*;
-
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.FileOutputStream;
@@ -64,6 +62,13 @@ import io.fabric8.maven.docker.util.ImageName;
 import io.fabric8.maven.docker.util.JsonFactory;
 import io.fabric8.maven.docker.util.Logger;
 import io.fabric8.maven.docker.util.TimestampFactory;
+
+import static java.net.HttpURLConnection.HTTP_CREATED;
+import static java.net.HttpURLConnection.HTTP_INTERNAL_ERROR;
+import static java.net.HttpURLConnection.HTTP_NOT_FOUND;
+import static java.net.HttpURLConnection.HTTP_NOT_MODIFIED;
+import static java.net.HttpURLConnection.HTTP_NO_CONTENT;
+import static java.net.HttpURLConnection.HTTP_OK;
 
 /**
  * Implementation using <a href="http://hc.apache.org/">Apache HttpComponents</a>
@@ -483,7 +488,7 @@ public class DockerAccessWithHcClient implements DockerAccess {
         JsonObject imageDetails = JsonFactory.newJsonObject(response.getBody());
 
         JsonElement repoTags = imageDetails.get("RepoTags");
-        if (repoTags.isJsonNull()) {
+        if (repoTags == null || repoTags.isJsonNull()) {
             return Collections.emptyList();
         }
 

--- a/src/main/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClient.java
+++ b/src/main/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClient.java
@@ -481,7 +481,13 @@ public class DockerAccessWithHcClient implements DockerAccess {
             return null;
         }
         JsonObject imageDetails = JsonFactory.newJsonObject(response.getBody());
-        JsonArray tagsArr = imageDetails.get("RepoTags").getAsJsonArray();
+
+        JsonElement repoTags = imageDetails.get("RepoTags");
+        if (repoTags.isJsonNull()) {
+            return Collections.emptyList();
+        }
+
+        JsonArray tagsArr = repoTags.getAsJsonArray();
         if (tagsArr.size() == 0) {
             return Collections.emptyList();
         }

--- a/src/test/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClientTest.java
+++ b/src/test/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClientTest.java
@@ -242,22 +242,39 @@ class DockerAccessWithHcClientTest {
 
     @Test
     void testRepoTagsIsNull() throws IOException {
-        String imagesId = "123123";
+        String imageId = "123123";
         ApacheHttpClientDelegate.HttpBodyAndStatus bodyAndStatus = new ApacheHttpClientDelegate.HttpBodyAndStatus(HTTP_OK, "{\"RepoTags\": null}");
 
         Mockito.doReturn(bodyAndStatus)
                 .when(mockDelegate)
                 .get(
-                        Mockito.eq(BASE_URL + "/v1.40/images/" + imagesId + "/json"),
+                        Mockito.eq(BASE_URL + "/v1.40/images/" + imageId + "/json"),
                         Mockito.any(ApacheHttpClientDelegate.BodyAndStatusResponseHandler.class),
                         Mockito.eq(HTTP_OK),
                         Mockito.eq(HTTP_NOT_FOUND)
                 );
 
-        List<String> imageTags = client.getImageTags(imagesId);
+        List<String> imageTags = client.getImageTags(imageId);
         Assertions.assertTrue(imageTags.isEmpty());
     }
 
+    @Test
+    void testNoRepoTagsInInspect() throws IOException {
+        String imageId = "123123";
+        ApacheHttpClientDelegate.HttpBodyAndStatus bodyAndStatus = new ApacheHttpClientDelegate.HttpBodyAndStatus(HTTP_OK, "{}");
+
+        Mockito.doReturn(bodyAndStatus)
+                .when(mockDelegate)
+                .get(
+                        Mockito.eq(BASE_URL + "/v1.40/images/" + imageId + "/json"),
+                        Mockito.any(ApacheHttpClientDelegate.BodyAndStatusResponseHandler.class),
+                        Mockito.eq(HTTP_OK),
+                        Mockito.eq(HTTP_NOT_FOUND)
+                );
+
+        List<String> imageTags = client.getImageTags(imageId);
+        Assertions.assertTrue(imageTags.isEmpty());
+    }
 
     private void givenAnImageName(String imageName) {
         this.imageName = imageName;

--- a/src/test/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClientTest.java
+++ b/src/test/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClientTest.java
@@ -16,7 +16,6 @@ import org.apache.commons.lang3.tuple.Pair;
 import org.apache.http.client.HttpResponseException;
 import org.apache.http.client.ResponseHandler;
 import org.apache.http.client.methods.CloseableHttpResponse;
-import org.apache.http.client.methods.HttpUriRequest;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
@@ -72,7 +71,7 @@ class DockerAccessWithHcClientTest {
     @BeforeEach
     public void setup() throws IOException {
         Mockito.doReturn("{\"ApiVersion\":\"1.40\",\"Os\":\"linux\",\"Arch\":\"amd64\"}")
-            .when(mockDelegate).get(BASE_URL + "/version", HTTP_OK);
+                .when(mockDelegate).get(BASE_URL + "/version", HTTP_OK);
 
         client = new DockerAccessWithHcClient(BASE_URL, null, 1, mockLogger) {
             @Override
@@ -241,6 +240,25 @@ class DockerAccessWithHcClientTest {
         Assertions.assertEquals("1.40", client.getServerApiVersion());
     }
 
+    @Test
+    void testRepoTagsIsNull() throws IOException {
+        String imagesId = "123123";
+        ApacheHttpClientDelegate.HttpBodyAndStatus bodyAndStatus = new ApacheHttpClientDelegate.HttpBodyAndStatus(HTTP_OK, "{\"RepoTags\": null}");
+
+        Mockito.doReturn(bodyAndStatus)
+                .when(mockDelegate)
+                .get(
+                        Mockito.eq(BASE_URL + "/v1.40/images/" + imagesId + "/json"),
+                        Mockito.any(ApacheHttpClientDelegate.BodyAndStatusResponseHandler.class),
+                        Mockito.eq(HTTP_OK),
+                        Mockito.eq(HTTP_NOT_FOUND)
+                );
+
+        List<String> imageTags = client.getImageTags(imagesId);
+        Assertions.assertTrue(imageTags.isEmpty());
+    }
+
+
     private void givenAnImageName(String imageName) {
         this.imageName = imageName;
     }
@@ -289,12 +307,12 @@ class DockerAccessWithHcClientTest {
         }
 
         Mockito.doReturn(array.toString())
-            .when(mockDelegate)
-            .get(Mockito.anyString(), Mockito.eq(HTTP_OK));
+                .when(mockDelegate)
+                .get(Mockito.anyString(), Mockito.eq(HTTP_OK));
     }
 
     private void givenThePushWillFailAndEventuallySucceed(final int retries) throws IOException {
-        failPost(retries-1);
+        failPost(retries - 1);
     }
 
     private void givenThePushWillFail(final int failures) throws IOException {
@@ -303,49 +321,49 @@ class DockerAccessWithHcClientTest {
 
     private void failPost(int retries) throws IOException {
         for (int i = 0; i < retries; ++i) {
-            Mockito.doThrow(new HttpResponseException(HTTP_INTERNAL_ERROR, "error-"+i))
-                .when(mockDelegate)
-                .post(Mockito.anyString(), Mockito.isNull(), Mockito.anyMap(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
+            Mockito.doThrow(new HttpResponseException(HTTP_INTERNAL_ERROR, "error-" + i))
+                    .when(mockDelegate)
+                    .post(Mockito.anyString(), Mockito.isNull(), Mockito.anyMap(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
         }
     }
 
     private void givenThePostWillFail() throws IOException {
         Mockito.doThrow(new HttpResponseException(HTTP_INTERNAL_ERROR, "error"))
-            .when(mockDelegate)
-            .post(Mockito.anyString(), Mockito.any(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
+                .when(mockDelegate)
+                .post(Mockito.anyString(), Mockito.any(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
     }
 
     @SuppressWarnings("unchecked")
     private void givenThatGetWillSucceedWithOk() throws IOException {
         Mockito.doReturn(HTTP_OK)
-            .when(mockDelegate)
-            .get(Mockito.anyString(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK), Mockito.eq(HTTP_NOT_FOUND));
+                .when(mockDelegate)
+                .get(Mockito.anyString(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK), Mockito.eq(HTTP_NOT_FOUND));
     }
 
     @SuppressWarnings("unchecked")
     private void givenThatGetWillSucceedWithNotFound() throws IOException {
         Mockito.doReturn(HTTP_NOT_FOUND)
-            .when(mockDelegate)
-            .get(Mockito.anyString(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK), Mockito.eq(HTTP_NOT_FOUND));
+                .when(mockDelegate)
+                .get(Mockito.anyString(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK), Mockito.eq(HTTP_NOT_FOUND));
     }
 
     private void givenTheGetWillFail() throws IOException {
         Mockito.doThrow(new HttpResponseException(HTTP_INTERNAL_ERROR, "error"))
-            .when(mockDelegate)
-            .get(Mockito.anyString(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
+                .when(mockDelegate)
+                .get(Mockito.anyString(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
     }
 
     private void givenTheGetWithoutResponseHandlerWillFail() throws IOException {
         Mockito.doThrow(new HttpResponseException(HTTP_INTERNAL_ERROR, "error"))
-            .when(mockDelegate)
-            .get(Mockito.anyString(), Mockito.eq(HTTP_OK));
+                .when(mockDelegate)
+                .get(Mockito.anyString(), Mockito.eq(HTTP_OK));
     }
 
     @SuppressWarnings("unchecked")
     private void givenThatDeleteWillSucceed() throws IOException {
         Mockito.doReturn(new ApacheHttpClientDelegate.HttpBodyAndStatus(HTTP_OK, "body"))
-            .when(mockDelegate)
-            .delete(Mockito.anyString(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK), Mockito.eq(HTTP_NOT_FOUND));
+                .when(mockDelegate)
+                .delete(Mockito.anyString(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK), Mockito.eq(HTTP_NOT_FOUND));
     }
 
     private void thenImageWasNotPushed() {
@@ -359,9 +377,9 @@ class DockerAccessWithHcClientTest {
     private void thenPushSucceeded(String imageNameWithoutTag, String tag) throws IOException {
         ArgumentCaptor<String> urlCapture = ArgumentCaptor.forClass(String.class);
         Mockito.verify(mockDelegate)
-            .post(urlCapture.capture(), Mockito.isNull(), Mockito.anyMap(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
+                .post(urlCapture.capture(), Mockito.isNull(), Mockito.anyMap(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
 
-            String expectedUrl = String.format("%s/v1.40/images/%s%%2F%s/push?force=1&tag=%s", BASE_URL, registry,
+        String expectedUrl = String.format("%s/v1.40/images/%s%%2F%s/push?force=1&tag=%s", BASE_URL, registry,
                 imageNameWithoutTag, tag);
         Assertions.assertEquals(expectedUrl, urlCapture.getValue());
     }
@@ -374,9 +392,9 @@ class DockerAccessWithHcClientTest {
     private void thenImageWasTagged(String imageNameWithoutTag, String tag) throws IOException {
         ArgumentCaptor<String> urlCapture = ArgumentCaptor.forClass(String.class);
         Mockito.verify(mockDelegate)
-            .post(urlCapture.capture(), Mockito.eq(HTTP_CREATED));
+                .post(urlCapture.capture(), Mockito.eq(HTTP_CREATED));
 
-            String expectedUrl = String.format("%s/v1.40/images/%s%%3A%s/tag?force=0&repo=%s%%2F%s&tag=%s", BASE_URL,
+        String expectedUrl = String.format("%s/v1.40/images/%s%%3A%s/tag?force=0&repo=%s%%2F%s&tag=%s", BASE_URL,
                 imageNameWithoutTag, tag, registry, imageNameWithoutTag, tag);
         Assertions.assertEquals(expectedUrl, urlCapture.getValue());
     }
@@ -467,16 +485,16 @@ class DockerAccessWithHcClientTest {
 
     private void givenPostCallThrowsException() throws IOException {
         Mockito.doThrow(new IOException("Problem with images/create"))
-            .when(mockDelegate)
-            .post(Mockito.anyString(), Mockito.any(), Mockito.anyMap(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
+                .when(mockDelegate)
+                .post(Mockito.anyString(), Mockito.any(), Mockito.anyMap(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
     }
 
     private void thenImageWasPulled() throws IOException {
         ArgumentCaptor<String> urlCapture = ArgumentCaptor.forClass(String.class);
         Mockito.verify(mockDelegate)
-            .post(urlCapture.capture(), Mockito.isNull(), Mockito.anyMap(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
+                .post(urlCapture.capture(), Mockito.isNull(), Mockito.anyMap(), Mockito.any(ResponseHandler.class), Mockito.eq(HTTP_OK));
 
-        String postUrl= urlCapture.getValue();
+        String postUrl = urlCapture.getValue();
         Assertions.assertNotNull(postUrl);
         Assertions.assertEquals("tcp://1.2.3.4:2375/v1.40/images/create?tag=1.1", postUrl);
     }

